docs/code/use_dataset_in_paddlepaddle.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Comment:
do not need period in commit message.

Reply:
Done.

Reply:
The commit message should be `docs: add paddlepaddle instructions in "INTEGRATIONS"` without period.